nilearn/tests/test_signal.py

```diff
@@ -359,6 +359,48 @@ def test_clean_t_r():
                 del det_one_tr, det_diff_tr
 
 
+def test_clean_kwargs():
+    """Providing kwargs to clean should change the filtered results."""
+    n_samples = 34
+    n_features = 501
+    x_orig = generate_signals_plus_trends(
+        n_features=n_features, n_samples=n_samples
+    )
+    kwargs = [
+        {
+            "butterworth__padtype": "even",
+            "butterworth__padlen": 100,
+            "butterworth__order": 3,
+        },
+        {
+            "butterworth__padtype": None,
+            "butterworth__padlen": None,
+            "butterworth__order": 0,
+        },
+        {
+            "butterworth__padtype": "constant",
+            "butterworth__padlen": 20,
+            "butterworth__order": 10,
+        },
+    ]
+    # Base result
+    t_r, high_pass, low_pass = 0.8, 0.01, 0.08
+    base_filtered = nisignal.clean(
+        x_orig, t_r=t_r, low_pass=low_cutoff, high_pass=high_cutoff
```

Comment:
Looks like `low_cutoff` and `high_cutoff` are undefined here. Did you mean `low_pass=low_pass, high_pass=high_pass`

Reply:
Ah yes good catch. Thanks!